gnd/src/main/java/com/google/android/gnd/persistence/remote/firestore/schema/FeatureDocument.java

```diff
@@ -27,7 +28,12 @@
   @Nullable private String customId;
   @Nullable private String caption;
   @Nullable private GeoPoint location;
-  @Nullable private String geoJson;
+  /**
+   * Use geometry instead.
```

Comment:
Was this meant to be a TODO? Please replace with `// TODO(#929): Harmonize representation of points, polygons, and geometries in remote db.`

Reply:
updated.